Makefile

```diff
@@ -12,3 +12,82 @@ build-arwen:
 
 test: clean build-arwen
 	go test -count=1 ./...
+
+build-c-contracts:
+	erdpy build ./test/contracts/erc20
+	erdpy build ./test/contracts/counter
+
+	erdpy build ./test/contracts/init-correct
+	erdpy build ./test/contracts/init-simple
+	erdpy build ./test/contracts/init-wrong
+	erdpy build ./test/contracts/misc
+	erdpy build ./test/contracts/signatures
+	erdpy build ./test/contracts/elrondei
+	erdpy build ./test/contracts/breakpoint
+
+	erdpy build ./test/contracts/exec-same-ctx-simple-parent
+	erdpy build ./test/contracts/exec-same-ctx-simple-child
+	erdpy build ./test/contracts/exec-same-ctx-child
+	erdpy build ./test/contracts/exec-same-ctx-parent
+	erdpy build ./test/contracts/exec-dest-ctx-parent
+	erdpy build ./test/contracts/exec-dest-ctx-child
+	erdpy build ./test/contracts/exec-same-ctx-recursive
+	erdpy build ./test/contracts/exec-same-ctx-recursive-parent
+	erdpy build ./test/contracts/exec-same-ctx-recursive-child
+	erdpy build ./test/contracts/exec-dest-ctx-recursive
+	erdpy build ./test/contracts/exec-dest-ctx-recursive-parent
+	erdpy build ./test/contracts/exec-dest-ctx-recursive-child
+	erdpy build ./test/contracts/async-call-parent
+	erdpy build ./test/contracts/async-call-child
+	erdpy build ./test/contracts/exec-same-ctx-builtin
```

Comment:
Some sci-fi feature would be very helpful here: `erdpy build --recursive ./test/contracts`. This is a very specific use case, though, and might not be needed.